lib/libc/gen/tls_malloc.c

```diff
@@ -101,6 +109,18 @@ union	overhead {
 #define	ov_index	ovu.ovu_index
 };
 
+struct pagepool_header {
+	size_t			 ph_size;
+#ifdef __CHERI_PURE_CAPABILITY__
+	size_t			 ph_pad1;
+#endif
+	struct pagepool_header	*ph_next;
+#ifdef CAPREVOKE
+	void			*ph_shadow;
+	void			*ph_pad2; /* Align strongly */
```

Comment:
To what?

Reply:
It pads the whole structure to 64-bytes. I don't recall what my motivation was at the time.